handlers/AdminHandlers/AdminGameHandlers.py

```diff
@@ -160,7 +160,7 @@ def post(self, *args, **kwargs):
             self.render("public/404.html")
 
     def add_source_code(self):
-        box = Box.by_uuid(self.get_argument('box_uuid'), '')
+        box = Box.by_uuid(self.get_argument('box_uuid'))
```

Comment:
I suggest give a default value when finding box_uuis argument. I guess the author's original idea is ```box = Box.by_uuid(self.get_argument('box_uuid', ''))```